paseos/tests/actor_builder_test.py

```diff
@@ -95,3 +95,19 @@ def test_add_comm_device():
     assert len(sat1.communication_devices) == 2
     assert sat1.communication_devices["dev1"].bandwidth_in_kbps == 10
     assert sat1.communication_devices["dev2"].bandwidth_in_kbps == 42
+
+
+def test_set_spacecraft_body_model():
+    """Check if we can set the spacecraft body model, and if the moments of inertia are calculated correctly"""
+    _, sat1, _ = get_default_instance()
+    ActorBuilder.set_spacecraft_body_model(sat1, mass=100)
+
+    assert sat1.mass == 100
+    assert all(
+        sat1.body_center_of_gravity == np.array([0, 0, 0])
```

Comment:
this is the default value I think, so you never actually check if the computation is correct.

Reply:
should check for another mesh